sigstore/_cli.py

```diff
@@ -692,3 +719,35 @@ def _get_identity_token(args: argparse.Namespace) -> Optional[str]:
             issuer,
         )
     return token
+
+
+class SplitCertificateChainError(Exception):
+    pass
+
+
+def _split_certificate_chain(chain_pem: str) -> List[bytes]:
+    PEM_BEGIN_CERTIFICATE = "-----BEGIN CERTIFICATE-----"
+
+    # Use the "begin certificate" marker as a delimiter to split the chain
+    certificate_chain = chain_pem.split(PEM_BEGIN_CERTIFICATE)
+
+    # Check for no certificates.
+    if not certificate_chain:
+        raise SplitCertificateChainError("empty PEM file")
```

Comment:
I think this won't actually catch the empty PEM file case, since `split` on an empty string always returns `['']`, not `[]`. Maybe test the `chain_pem` instead?

```suggestion
    # Check for no certificates.
    if not chain_pem:
        raise SplitCertificateChainError("empty PEM file")
```

Reply:
[ce3dc89](https://github.com/sigstore/sigstore-python/pull/323/commits/ce3dc8921a065472b2583784a49604f428fb0135)